app/src/main/java/com/example/topmovies/fragment/AboutDialogFragment.kt

```diff
@@ -12,8 +12,7 @@ class AboutDialogFragment : DialogFragment() {
         return AlertDialog.Builder(requireActivity())
             .setTitle(getString(R.string.about_dialog_fragment_title))
             .setMessage(getString(R.string.about_dialog_fragment_message))
-            .setPositiveButton(getString(R.string.ok))
-            { _, _ -> }
+            .setPositiveButton(getString(R.string.ok)) { _, _ -> }
```

Comment:
Why this one is edited?

Reply:
I got it, “never touch a code, which don't belong to your issue”